frontend/src/components/details/NavbarComponent.js

```diff
@@ -12,37 +12,44 @@ import { Link } from 'react-router-dom';
 const NavbarComponent = (props) => {
     if (!props.loadComparePage) {
         return (
-            <div className='navbar w3-row' vertical='center' horizontal='space-between'>
+            <div className='navbar' vertical='center' horizontal='space-between'>
                 <Link to={{ pathname: slugs.categories }}>
-                    <btn className='w3-col l1 m1 s1 w3-center'>
+                    <button className='w3-center'>
                         <i class='fa fa-chevron-left' aria-hidden='true' />
-                    </btn>
+                    </button>
                 </Link>
-                <b className='w3-col l6 m6 s4'>{props.scenarioName}</b>
+                <b>{props.scenarioName}</b>
 
-                <Link to={{ pathname: slugs.details }} onClick={props.onExportClicked}>
-                    <pdfbtn className='w3-col l3 m3 s4'>
-                        <i className='fa fa-file-pdf-o w3-margin-right' aria-hidden='true'></i>
-                        Export Pdf
+                <Link to={{ pathname: slugs.details }} onClick={props.onExportClick}>
```

Comment:
I suggest refactoring this Nav bar. The buttons are way too wide in my opinion.

![image](https://user-images.githubusercontent.com/22003632/122567829-645df000-d049-11eb-9751-fa1902f59a94.png)

![image](https://user-images.githubusercontent.com/22003632/122567864-6de75800-d049-11eb-9902-062e116f6704.png)

Only the Name and Icon should be linking, not the whole whitespace.

Reply:
I agree. Do you want to change the width of the element or only the area that is clickable?

Reply:
I am not sure about that. But I would attach the Export and the Add button to the right side, so that they don't resize, but move when the Window gets resized. and attach the back button and the Name of the Scenario to the left, and make the bar take up the empty space in the middle between those two groups.